packages/react-components/react-combobox/src/utils/useComboboxBaseState.ts

```diff
@@ -3,12 +3,12 @@ import { useControllableState, useFirstMount } from '@fluentui/react-utilities';
 import { useOptionCollection } from '../utils/useOptionCollection';
 import { OptionValue } from '../utils/OptionCollection.types';
 import { useSelection } from '../utils/useSelection';
-import type { ComboboxBaseProps, ComboboxBaseOpenEvents } from './ComboboxBase.types';
+import type { ComboboxBaseProps, ComboboxBaseOpenEvents, ComboboxBaseState } from './ComboboxBase.types';
 
 /**
  * State shared between Combobox and Dropdown components
  */
-export const useComboboxBaseState = (props: ComboboxBaseProps) => {
+export const useComboboxBaseState = (props: ComboboxBaseProps, editable = false): ComboboxBaseState => {
```

Comment:
why is editable not in props?

Reply:
it's not part of user-provided props. Combobox is editable, but Dropdown isn't, so it's baked into the choice of component. I can expand the type of the argument to be `props: ComboboxBaseProps & { editable?: boolean; }` though